tools/resource-gen/genutils/util.go

```diff
@@ -93,6 +97,16 @@ func ToResourceInfo(desc protoreflect.MessageDescriptor) ResourceInfo {
 			}
 		}
 	}
+	out.IsPolicy = !out.SkipRegistration && !out.Global && !out.WsAdminOnly && !out.WsReadOnly && out.ResourceType != "Dataplane"
```

Comment:
Why is `Global` or `ReadOnly` not a policy?

Reply:
Cause all policies depend of a Mesh. ReadOnly is only used for internal purpose

Reply:
Adding a comment for this